### ISSUE TITLE
[Feature Request]: Layouts from LSSM v3 (with improvments)

### ISSUE THREAD

**Issue by NOEcom** [LSS-Manager/LSSM-V.4#372]
**Is your feature request related to a problem?**
Currently LSSM v3 is still required when wanting to use a custom Layout. They are within the AppStore as a module for each. 

**Solution / Feature**
I would suggest adding a module to LSSM v4 called "Layout Editor" which allows the user to choose from a template for there game layout or create a custom layout through the edit. (I understand that this is a bigger project) For the beginning the currantly available layouts within LSSM v3 could serve well as templates while requiring some improvement (Example: Layout 1 has a weird blank space underneath the layouted area which causes the layout to move out of view when scrolling through the Missions from time to time. 
Functions for sharing custom layouts would also be great. 

**Alternatives**
If an Editor is to big of a scope then simply adding the LSSM v3 Layouts with slight improvement should serve well. 

**Additional**
Example of issues of Layout 1: https://i.imgur.com/qruRCZV.jpg

**Comment by jxn-30** [comment]
Yes, it is planned to create an Layout editpr but as mentioned totally right, this is very much work to do 😉
